capsules/src/led.rs

```diff
@@ -7,20 +7,20 @@ use kernel::{AppId, Driver, ReturnCode};
 use kernel::hil;
 
 /// Whether the LEDs are active high or active low on this platform.
+#[derive(Clone,Copy)]
 pub enum ActivationMode {
     ActiveHigh,
     ActiveLow,
 }
 
 pub struct LED<'a, G: hil::gpio::Pin + 'a> {
-    pins: &'a [&'a G],
-    mode: ActivationMode,
+    pins_init: &'a [(&'a G, ActivationMode)],
```

Comment:
That would be super nice indeed. I'll add that to the wiki so that we know it's an easy to-do item for later.